docs/core/integration-quality-scale/rules/iqs045.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "IQS045 - The documentation describes all integration configuration options"
+---
+
+IQS045 is as follows:
+
+The documentation describes all integration configuration options
+
+## Reasoning
+
+Integrations can provide an options flow to allow users to change integration configuration.
+This rule ensures that all configuration options are documented so that users can understand what each option does and how to use it.
+
+## Example implementation
+
+In the following example, the integration has an options flow where the user can configure if the integration should add light groups as entities.
+
+```markdown
+## Configuration options
+
+The integration provides a single setting, you are able to disable the creation of light groups as entities.
+```
+
```

Comment:
```suggestion
The following example is for an integration with multiple configuration options, using the 

{% configuration_basic %}
  Host:
    description: Hostname or IP address of your Nuki bridge. For example, `192.168.1.25`.
  Port:
    description: Port of the Nuki bridge HTTP API. Default is `8080`.
  Token:
    description: Token to authenticate against the Nuki bridge HTTP API.
  Use an encrypted token for authentication:
    description: Use an encrypted token for API calls to the bridge. This should only be deactivated if you experience issues with the API (such as authentication). The default is `True`.
{% endconfiguration_basic %}
```

Reply:
or, another example could be

{% configuration_basic %}
Username:
  description: "Your username you use to log into the La Marzocco app."
  required: true
  type: string
Password:
  description: "The password you use to log into the La Marzocco app."
  required: true
  type: string
Host:
  description: "The IP address of your machine in your local network. If not set, no local connections will be used."
  required: false
  type: boolean
{% endconfiguration_basic %}

Reply:
I chose the first one, as the second one mentioned a list, and that's only for YAML.